src/styles/blocks/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  gap: $calendar-gep;
+  flex-basis: wigth / 7;
+  width: $day-width * 7 + $calendar-padding * 2 + $calendar-gep * 6;
+  padding: $calendar-padding;
+
+  @each $day, $number in $day-of-week {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($number - 1) * ($day-width + $calendar-gep);
+    }
+  }
+
+  @for $last-day from 28 through 31 {
+    &--month-length--#{$last-day} :nth-child(n + #{$last-day + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $day-width;
+    height: $day-height;
+    border: 1px solid black;
+    transition: background-color 0.5s transform 0.5s;
```

Анімація не працює...  
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```

---

Як не дивно в мене працює анімація і без коми, але додала) Дякую